notifier/typetalk/client.go

```diff
@@ -0,0 +1,72 @@
+package typetalk
+
+import (
+	"errors"
+	"os"
+	"strconv"
+
+	"github.com/mercari/tfnotify/terraform"
+	typetalk "github.com/nulab/go-typetalk/typetalk/v1"
+)
+
+// EnvToken is Typetalk API Token
+const EnvToken = "TYPETALK_TOKEN"
+
+// Client represents Typetalk API client.
+type Client struct {
+	*typetalk.Client
+	Config Config
+	common service
+	Notify *NotifyService
+	API    API
+}
+
+// Config is a configuration for Typetalk Client
+type Config struct {
+	Token    string
+	Title    string
+	TopicID  string
+	Message  string
+	CI       string
+	Parser   terraform.Parser
+	Template terraform.Template
+}
+
+type service struct {
+	client *Client
+}
+
+// NewClient returns Client initialized with Config
+func NewClient(cfg Config) (*Client, error) {
+	token := cfg.Token
```

Comment:
```suggestion
	token := os.ExpandEnv(cfg.Token)
```

It might be better to use ExpandEnv function.

Reply:
@b4b4r07 Thanks! I updated it.